server/src/internalClusterTest/java/org/opensearch/search/fetch/subphase/highlight/HighlighterSearchIT.java

```diff
@@ -127,6 +129,7 @@
 import static org.hamcrest.Matchers.not;
 import static org.hamcrest.Matchers.startsWith;
 
+@TimeoutSuite(millis = 35 * TimeUnits.MINUTE)
```

Comment:
This seems like a really long time.  Is there any insight you can provide for why it should take this long?

Reply:
I think this is mostly a function of how many tests are squeezed into this 1 class.
The suite has ~90 tests, which turns into ~180 when parameterized with concurrent search. Looking at the logs for several of the failures ([ref](https://github.com/opensearch-project/OpenSearch/issues/12119#issuecomment-1922017503)), we see that the first pass (either concurrent search disabled or enabled) takes around 14 minutes, which averages out to ~10 seconds per test. The is corroborated by the [junit report](https://build.ci.opensearch.org/job/gradle-check/32902/testReport/junit/org.opensearch.search.fetch.subphase.highlight/HighlighterSearchIT/) as well.

Looking at what the tests are actually doing, most of them are actually creating an index and then indexing docs and then searching instead of re-using indices across tests, so 10 seconds per test does not seem unreasonable to me.

Reply:
Please add that comment to the code, so the context is preserved

Reply:
Added a comment and a link to the GitHub issue